### PR TITLE
DAN-59 updated Mapbox to use new endpoint

### PR DESCRIPTION
**Pull request by sammccants**
This change updates the Mapbox integration example to use the new /catalog endpoint of the Catalog API, making the X/Y tile calculations no longer necessary. It also adds a couple comments in the Bing integration.

### CODE REVIEW

**Review by ejke — APPROVED** [review]

**Review by austintino — APPROVED** [review]

File: Mapbox/index.html [hunk 1]
```diff
@@ -1,254 +1,244 @@
...
+				// however, you could extend this functionality to use any of the available images,
+				// allow the user to switch between multiple images, etc
+				*/
+				console.log(catalogResponse);
+				let firstCatalogItem = catalogResponse['items'][0];
+				/*
+				// Get the x and y coordinates of the upper-left and bottom-right tiles
+				// from the available 5x5 grid
+				*/
+				const minTileX = catalogResponse['minTileX'];
```
Comment (charliemoriarty) [inline Mapbox/index.html]: You could do some sweet object shorthand here if you wanted to:
```
const {minTileX, minTileY, maxTileX, maxTileY} = catalogResponse
```
(I think that's it)

**Review by charliemoriarty — APPROVED** [review]
Woohoo!

File: Mapbox/index.html (hunk 1 above)
Comment (sammccants) [reply Mapbox/index.html]: It's so beautiful! I've changed it in both of these, and I'll do the same in the others when I get to them!

**Comment by charliemoriarty** [comment]
@sam-birdi 
beast

**Pull request merged**